<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttandenceServer</class>
 <widget class="QMainWindow" name="AttandenceServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AttandenceServer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
font: 10pt &quot;微软雅黑&quot;;
}
QMainWindow{
background-color: rgba(49, 49, 49,0.8);
}
QLabel{
border : none;
border-radius:15px;
	
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>251</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 250,0, 234);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="registerBt">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>400</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>员工注册</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>24</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="sexEdit">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>84</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="addressEdit">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>144</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="phoneEdit">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>204</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="birthdayEdit">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>264</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="kaoqingEdit">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>324</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
